backend/tests/test_auth.py

```
@@ -0,0 +1,108 @@
+            "password": "testpass"
+        }
+    )
+    assert response.status_code == 200
+    json_data = response.json()
+    assert "access_token" in json_data
+    assert json_data["token_type"] == "bearer"
+
+# ----------------------
+# ✅ Test fetch current user
+# ----------------------
+def test_get_current_user(test_user):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_get_current_user`

Add return type annotation: `None`

(ANN201)